docs/core/diagnostics/index.md

```diff
@@ -0,0 +1,27 @@
+---
+title: .NET Core Diagnostics
+description: An overview of the tools and APIs available to diagnose .NET Core applications.
+author: stmaclea
+ms.author: stmaclea
+ms.date: 8/5/2019
+ms.topic: article
+ms.prod: .net-core
+---
+# .NET Core Diagnostics
```

Comment:
Please review the guidelines for an overview article here: https://review.docs.microsoft.com/en-us/help/contribute/contribute-how-to-write-overview?branch=master and see if you can structure it accordingly. These are general guidelines since this is not an "Azure Service" as the guideline was really written for. But it's a nice template and we're trying to conform (where it makes sense) to it.

Reply:
I'm not sure overview is the correct template....  It is intended as an index or local TOC.

I revised it to match the style of the https://docs.microsoft.com/en-us/dotnet/core/additional-tools/ overview.

Reply:
As a writing org we're moving away from index/local toc style articles and to overviews. The additional-tools article is a carryover from the old way.

Reply:
I spent a lot of time looking at this. The .NET Core Guide does not have any good overview articles.  It looks like the Machine Learning and Spark sections are more up to date and are better references.

I seriously considered removing the overview altogether, but I think it is a good place to provide some guidance.

I improved this a bit more by changing the article title and adding a customer intent comment.

We have a few more PRs which will touch this section and this overview. I think this is a good place to start.